core/plugins/org.polarsys.capella.core.libraries.ui/src/org/polarsys/capella/core/libraries/ui/handlers/ManageReferencedLibrariesHandler.java

```diff
@@ -47,6 +51,8 @@
 import org.polarsys.capella.core.model.handler.command.CapellaResourceHelper;
 
 public class ManageReferencedLibrariesHandler extends AbstractHandler {
+	
+	private static String LIBRARY_DETACHMENT_DOC_URI= "org.polarsys.capella.re.doc/html/09.%20Libraries/9.5.%20Advanced%20Operations.html?cp=6_1_20_4";
```

Comment:
can be good to have a junit looking if the page still exist.
can you look at https://github.com/eclipse/capella/blob/master/tests/plugins/org.polarsys.capella.test.doc.ju/src/org/polarsys/capella/test/doc/ju/BrokenLinksCheckTestCase.java